stanford_publication.libraries.yml

```diff
@@ -0,0 +1,12 @@
+
+node:
+  version: VERSION
+  css:
+    component:
+      dist/css/stanford_publication.node.css: {}
+
+view:
+  version: VERSION
+  css:
+    component:
+      dist/css/stanford_publication.views.css: {}
```

Comment:
Where do these get attached?

Reply:
Ah I see now.

Reply:
Oh I dont think I need this anymore. I did the same dynamic thing that I did in Stanford profile styles